src/akamai/js/akamaiHD_flash.js

```diff
@@ -242,6 +243,9 @@ require("../../../html5-common/js/utils/constants.js");
      * @param {boolean} rewind True if the stream should be set to time 0
      */
     this.load = function(rewind) {
+      if (loaded && !rewind) return;
+      this.callToFlash("load("+rewind+")");
+      loaded = true;
```

Comment:
Resetting it to false in this.setVideoUrl. Since setVideoUrl()  implementation is not part of this ticket, As of now I have added comment here that it should be set to false only if URL changes